x/tokenomics/keeper/settle_pending_claims.go

```diff
@@ -305,15 +331,97 @@ func (k Keeper) proofRequirementForClaim(ctx sdk.Context, claim *prooftypes.Clai
 	}
 
 	logger.Info(fmt.Sprintf(
-		"claim does not require proof due to compute units (%d) being less than the threshold (%d) and random sample (%.2f) being greater than probability (%.2f)",
-		numClaimComputeUnits,
+		"claim does not require proof due to claimed amount (%s) being less than the threshold (%s) and random sample (%.2f) being greater than probability (%.2f)",
+		claimeduPOKT,
 		proofParams.GetProofRequirementThreshold(),
 		proofRequirementSampleValue,
 		proofParams.GetProofRequestProbability(),
 	))
 	return requirementReason, nil
 }
 
+// slashSupplierStake slashes the stake of a supplier slashingCount times and mints
+// the total slashing amount to the tokenomics module account.
+func (k Keeper) slashSupplierStake(
+	ctx sdk.Context,
+	supplierOperatorAddress string,
+	slashingCount uint64,
+) error {
+	logger := k.logger.With("method", "slashSupplierStake")
+
+	proofParams := k.proofKeeper.GetParams(ctx)
+	slashingPenalty := proofParams.GetProofMissingPenalty()
+
+	totalSlashingAmt := slashingPenalty.Amount.Mul(math.NewIntFromUint64(slashingCount))
+	totalSlashingCoin := sdk.NewCoin(volatile.DenomuPOKT, totalSlashingAmt)
+
+	supplierToSlash, supplierFound := k.supplierKeeper.GetSupplier(ctx, supplierOperatorAddress)
+	if !supplierFound {
+		return types.ErrTokenomicsSupplierNotFound
+	}
+
+	remainingStakeAmt := math.NewInt(0)
+	if supplierToSlash.GetStake().Amount.GT(totalSlashingAmt) {
+		remainingStakeAmt = supplierToSlash.GetStake().Amount.Sub(totalSlashingAmt)
+	}
+
+	err := k.bankKeeper.MintCoins(ctx, types.ModuleName, sdk.NewCoins(totalSlashingCoin))
```

Comment:
s/types/tokenomictypes to make sure there's no ambiguity

Reply:
Instead of minting, what we need to do is move tokens from the supplier module to the tokenomics module. Otherwise accounting doesn't work.

#PUC at the top as well